MH4U Dex/RegionContainerViewController.m

```diff
@@ -0,0 +1,46 @@
+//
+//  RegionContainerViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/6/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "RegionContainerViewController.h"
+
+#import "RegionItemTableViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "CoreDataController.h"
+
+//NSString *const RegionNameKey = @"name";
+
+@interface RegionContainerViewController ()
+
+@property (nonatomic, weak) IBOutlet UIView *regionMapSubview;
+@property (nonatomic, weak) IBOutlet UIView *regionDropSubview;
+@property (nonatomic, weak) IBOutlet UILabel *regionNameLabel;
+
+@end
+
+@implementation RegionContainerViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    //[self.regionMapSubview setHidden:YES];
```

Comment:
Then add at least a comment-note to that effect and perhaps a `TODO:`, depending on why it's commented out and when/why it'll be re-added.